application/src/test/java/org/opentripplanner/updater/trip/FlexTripInput.java

```diff
@@ -0,0 +1,41 @@
+package org.opentripplanner.updater.trip;
```

Comment:
Sort of a weird package for this as we don't have support for flex in trip updates.